### PR TITLE
add bioconductor

### PR DESCRIPTION
**Pull request by orchid00**
to R and Bio
hi @jspaaks please review this PR.
thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Institute of Technology overview of software packages.
 * R
   * [CRAN](https://cran.r-project.org)
   * [rOpenSci](https://ropensci.org/packages/)
+  * [Bioconductor](https://www.bioconductor.org/)
```
Comment (jspaaks) [inline README.md:78]:
```suggestion
  * [Bioconductor](https://www.bioconductor.org/packages/release/BiocViews.html#___Software)
```

File: README.md [hunk 2]
```diff
@@ -113,6 +114,7 @@ Institute of Technology overview of software packages.
   * [bio.tools](https://bio.tools) - Essential scientific and technical
   information about software tools, databases and services for bioinformatics
   and the life sciences.
+  * [Bioconductor](https://www.bioconductor.org/)
```
Comment (jspaaks) [inline README.md:117]:
```suggestion
  * [Bioconductor](https://www.bioconductor.org/packages/release/BiocViews.html#___Software)
```

**Comment by jspaaks** [comment]
Hi @orchid00 and thanks for making the PR! Sorry I took so long to review it. 

After taking a quick look at the link, it seems that bioconductor is something that users need to install locally, correct? If so, that puts it outside of the scope of the AwesomeList. However, there is still relevant content accessible online at https://www.bioconductor.org/packages/release/BiocViews.html#___Software, so I've suggested changes in the PR to change the links to point to that specific part of the bioconductor website instead. Let met know what you think!

Best regards,
Jurriaan

**Comment by orchid00** [comment]
Hi Jurriaan, 

I missed this.
Actually Bioconductor is like a package manager, like PyPi 
When people install Bioconductor they have access to install directly Bioinformatics packages
i'll update the link
https://www.bioconductor.org/packages/release/BiocViews.html#___Software

File: README.md (hunk 2 above)
Comment (orchid00) [inline README.md:117]:
```suggestion
  * [Bioconductor](https://www.bioconductor.org/packages/release/BiocViews.html#___Software)
```

**Review by orchid00 — COMMENTED** [review]
all updated

**Comment by jspaaks** [comment]
Nice, thanks for being so patient!

**Pull request merged**